### PR TITLE
Nes 8 frontend skeleton

### PR DESCRIPTION
**Pull request by Tokushiro**
Not a completely finished but a good start

### CODE REVIEW

**Comment by nicki419** [comment]
The Solution file should be in the root of the working tree, and reference all projects thereon.

**Comment by nicki419** [comment]
The Layout folder should get its own _Imports.razor.

File: Famicon/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "http": {
+        "commandName": "Project",
+        "dotnetRunMessages": true,
+        "launchBrowser": true,
+        "applicationUrl": "http://localhost:5016",
+        "environmentVariables": {
+          "ASPNETCORE_ENVIRONMENT": "Development"
+        }
+      },
+      "https": {
```
Comment (nicki419) [inline Famicon/Properties/launchSettings.json:21]: This should be the default listener as opposed to the Http one. It should be moved to the top of the file.

**Review by nicki419 — APPROVED** [review]
Nicely done. Resolve the comments, then it can be merged.

File: Famicon/Properties/launchSettings.json (hunk 1 above)
Comment (Tokushiro) [reply Famicon/Properties/launchSettings.json:21]: Move it to the top

**Pull request merged**